src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $my-var;
+  gap: $width-gap;
+
+  .calendar__day {
+    background-color: #eee;
+    border: 1px solid #000;
+    width: 100px;
+    height: 100px;
```

Comment:
The height of the day box is also hardcoded. Consider using the same variable `$calendar-day-size` for both width and height, to ensure all day boxes are squares.